api/services/connect/workflows/paths/index.yaml

```
@@ -0,0 +1,17 @@
+get:
+  operationId: get_workflow_list
+  parameters:
+  - $ref: "../parameters/workspace_id.yaml"
+  - $ref: "../parameters/owner.yaml"
+  responses:
+    '200':
+      content: {}
+      description: Success
+    '400':
+      content: {}
+      description: Bad Request
+  summary: return a list of workflows by fetching all user workflows documents from
```

The API consumer doesn't need to know about where the documents are coming from or whether there's a DB involved.
```suggestion
  summary: Return all user workflows for a given workspace
```

---

Done.